dev-client/src/components/sites/CreateSiteView.tsx

```diff
@@ -55,25 +53,24 @@ type LocationInputOptions = 'coords' | 'gps' | 'pin';
 
 export default function CreateSiteView({
   defaultProject,
-  userLocation,
   createSiteCallback,
   sitePin,
 }: Props) {
   const {t} = useTranslation();
 
+  const userLocation = useSelector(state => state.map.userLocation);
```

Comment:
comment: Mostly just curious, is there a reason why you moved the selector down to this component? I prefer to pass the location down as a prop, and have the screen be the one pulling in the global state with a selector.

Reply:
i usually default to pushing hook usages as far down as possible if they can trigger a re-render and to avoid repetitive parameter passing, but it's one of those weird squishy intuitive things i think